src/util/FcmListener.js

```diff
@@ -228,7 +228,7 @@ async function pairingServer(client, guild, full, data, body) {
         switchGroups: server ? server.switchGroups : {},
         messageId: (message !== undefined) ? message.id : null,
         battlemetricsId: battlemetricsId,
-        connect: (info === null) ? 'Unavailable' : `${client.intlGet(guild.id, 'connect')} ${info.ip}:${info.port}`,
```

Comment:
You can remove these changes and replace 'Unavailable' with null.

Reply:
I am not quite sure what you mean by that :/